### PR TITLE
Disable apple device testing

### PR DESCRIPTION
**Pull request by mmitche**
https://github.com/dotnet/arcade/issues/13250 to figure out a longer term solution to this

### To double check:

* [ ] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/arcade/tree/main/Documentation/Validation

### CODE REVIEW

**Comment by mmitche** [comment]
I think we should move all the xharness testing to a separate optional PR job.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -191,35 +191,35 @@ stages:
...
-              -test
-              -warnAsError false
-              -projects $(Build.SourcesDirectory)/tests/XHarness.Apple.DeviceTests.proj
-              /bl:$(Build.SourcesDirectory)/artifacts/log/$(_BuildConfig)/Helix.XHarness.Apple.Device.Tests.binlog
-              /p:RestoreUsingNuGetTargets=false
-            displayName: XHarness Apple Device Helix Testing
-            env:
-              SYSTEM_ACCESSTOKEN: $(System.AccessToken)
-              HelixAccessToken: ''
+        # - job: Apple_Devices
```
Comment (riarenas) [inline azure-pipelines.yml:194]: Should we just delete instead?

**Review by dougbu — APPROVED** [review]

**Comment by dougbu** [comment]
Suggest this needs to be backported to at least release/6.0 and release/7.0. Haven't checked whether release/3.1 needs the same treatment&hellip;

**Comment by mmitche** [comment]
Just available resources.

**Pull request merged**